style.css

```diff
@@ -673,6 +516,160 @@ textarea {
     margin-bottom: 1rem;
   }
 
+  .popup-window {
+    box-sizing: border-box;
+
+    /* Auto layout */
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 12px;
+    position: absolute;
+    width: 343px;
+    height: 713px;
+    left: calc(50% - 343px / 2);
+    top: 60px;
+
+    /* Neutrals/N0 */
+
+    background: #fff;
+
+    /* Neutrals/N40 */
+
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .frame1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: 100%;
+    height: 40px;
+  }
+
+  .closeIcon-popup {
+    margin-left: 210px;
+    margin-bottom: 4rem;
+    color: #67798e;
+  }
+
+  .frame2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: 100%;
+    height: 32px;
+  }
+
+  .popup-img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .popups-project-p-desktop {
+    display: none;
+  }
+
+  .right-block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    width: 80%;
+  }
+
+  .action {
+    width: 100%;
+    margin: auto;
+    margin-left: 25px;
+    margin-top: 3px;
+  }
+
+  .overlay {
+    position: absolute;
+    left: 0;
+    width: auto;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    background: #c1c7d0;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(3px);
+    z-index: -1;
+  }
+
+  .seeSource {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 5.2px;
+    gap: 10px;
+    position: absolute;
+    width: 136px;
+    height: 48px;
+    left: 180px;
+  }
+
+  .seeLive {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 15px;
+    gap: 10px;
+    position: absolute;
+    width: 136px;
+    height: 48px;
+    left: 30px;
+  }
+
```

Comment:
- [ ] Kindly add the hover interaction to the buttons in the modal window to be similar to the Figma template 👇🏽 

> Please add the below interactions to the pop-up(model) buttons.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/209444483-36be3e92-a297-45de-8f67-b5364e5648dc.png) | ![image](https://user-images.githubusercontent.com/58642949/209444502-6d8424d9-4bb4-4881-ae1a-2225f1c75b9f.png) |